ocis-pkg/service/grpc/client.go

```diff
@@ -28,14 +28,14 @@ type ClientOptions struct {
 // Option is used to pass client options
 type ClientOption func(opts *ClientOptions)
 
-// WithTLSMode allows to set the TLSMode option for grpc clients
+// WithTLSMode allows setting the TLSMode option for grpc clients
```

Comment:
Note: these kind of sentences can also be reworded to put the "to set" at the end, in the passive:

`WithTLSMode allows the TLSMode option for grpc clients to be set`

Either read OK to me, so maybe we just do the simple change that I have done here - "to set" => "setting"